sdk/src/test/java/com/mapbox/maps/MapViewTest.kt

```diff
@@ -67,6 +67,13 @@ class MapViewTest {
   @Test
   fun getMapboxMap() {
     mapView.getMapboxMap()
+    verify { mapController.onDestroy() }
```

Comment:
kinda surprised why is it even called here...
guess due to some mocking but perhaps we can add more details why is it called here

Reply:
In this case, it is indeed because the objects are mocked and the Activity is null.  Not a great test, but it does behave as expected in this case. Maybe should add some kind of comments.